Assets/GothicVR/Scripts/Creator/WorldCreator.cs

```diff
@@ -36,6 +36,7 @@ public async Task CreateAsync(string worldName)
             DebugAnimationCreator.I.Create(worldName);
             DebugAnimationCreatorBSFire.I.Create(worldName);
             DebugAnimationCreatorVelaya.I.Create(worldName);
+            DebugAnimationCreatorBloodwyn.I.Create();
 
```

Comment:
flag for NPC to spawn is set but the VM function to spawn the npcs in OC is not here
```suggestion
            if(FeatureFlags.I.CreateOcNpcs)
                PxVm.CallFunction(GameData.I.VmGothicPtr, "STARTUP_OLDCAMP");
```